spec/rails_helper.rb

```diff
@@ -50,4 +52,6 @@
 
   config.include AutenticationHelpers, type: :controller
   config.include ActionDispatch::TestProcess
+
+  OmniAuth.config.test_mode = true
```

Comment:
I think this config should be added in the https://github.com/tspenov/maycamp_arena/tree/master/spec/support folder with a proper name (`omniauth.rb`?)